content/influxdb/v2.0/upgrade/v1-to-v2.md

```diff
@@ -12,33 +12,181 @@ aliases:
+
+### Supported protocols
+
+If you're using one of the [supported protocols](/influxdb/v1.8/supported_protocols/) in InfluxDB 1.x (CollectD, Graphite, OpenTSDB, Prometheus, UDP),
+InfluxDB 2.0 doesn't support these protocols directly.
+Use [Telegraf](/telegraf/v1.16/) to translate from the source protocol to InfluxDB 2.0.
+
+### Kapacitor
+
+You can continue to use Kapacitor with InfluxDB OSS 2.0 under the following scenarios:
+
+- Kapacitor Batch-style TICKscripts work with the 1.x read compatible API
+  Existing Kapacitor user credentials should continue to work using the [1.x compatibility API](/influxdb/v2.0/reference/api/influxdb-1x/).
+- Kapacitor Stream-style TICKscripts will not work using the [subscription API](/influxdb/v1.8/administration/subscription-management/).
+  (There is no subscriptions API in InfluxDB 2.0.)
+  We recommend writing data directly to *both* Kapacitor and InfluxDB to allow stream tasks to continue to work.
+  To do this, configure two [InfluxDB output plugins](/telegraf/v1.16/plugins/#influxdb): one for Kapacitor and one for InfluxDB 2.0.
+  If you use other data sources to send data to InfluxDB, Telegraf can be used as an intermediate layer to write to both Kapacitor and InfluxDB.
+  <!-- 
+  Other architectures and mechanisms are possible, too.
+  Need more help? Reach out in the [community forums](https://community.influxdata.com).
+  -->
+
+### User migration
+
+`influxd upgrade` migrates existing 1.x users and their permissions.
+However, it *does not migrate administrative users*.
+
+To review users with admin permissions, in the InfluxDB 1.x CLI, run `show users`.
+Any users labeled "admin" *will not* be migrated.
+
+If you're using an admin user for visualization or Chronograf's administrative functions, you should create a new read-only user before upgrading.
+Admin rights are granted to the primary user created in the InfluxDB 2.0 setup process which runs at the end of the upgrade process.
+(This provides you with the opportunity to re-assess who should be granted admin-level access in your InfluxDB 2.0 setup.)
+
+### Dashboards
+
+You can continue to use your existing dashboards and visualization tools with InfluxDB 2.0 via the [1.x read compatibility API](/influxdb/v2.0/reference/api/influxdb-1x/).
+The upgrade process creates [DBRP mappings](/influxdb/v2.0/reference/api/influxdb-1x/dbrp/) to ensure existing users can execute InfluxQL queries with the appropriate permissions.
+
+However, if your dashboard tool is configured using a user with admin permissions,
+you will need to create a new read-only user with the appropriate database permissions *before* upgrading.
+This new username and password combination should be used within the data source configurations to continue to provide read-only access to the underlying data.
+
+Ensure your dashboards are all functioning before upgrading.
+
+### Other data
+
+The 1.x `_internal` database is not migrated with the `influxd upgrade` command.
+
```

Comment:
You may add something to this effect here:

To collect, store, and monitor similar internal InfluxDB metrics, [create an InfluxDB 2.0 scraper](/influxdb/v2.0/write-data/no-code/scrape-data/manage-scrapers/create-a-scraper/) to scrape data from the `/metrics` endpoint and store them in a bucket.